src/components/OfferCheckoutCard/OfferCheckoutCard.js

```diff
@@ -28,34 +28,30 @@ const OfferCheckoutCard = ({ isDataLoaded, t }) => {
     freePeriods,
     freeDays,
     expiresAt,
-    startTime
+    startTime,
+    customerPriceInclTax: grossPrice
```

Comment:
let's take price from order - it's our source of truth.
So lets keep 
```
const {
    priceBreakdown: { offerPrice },
    taxRate,
    country,
    currency
  } = useSelector(state => state.order.order);
  ```
Tax rate is used when we don't have a gross price (eg. trial), bo let's calculate it based on order - as it was